src/Alpha_complex/doc/Intro_alpha_complex.h

```diff
@@ -146,6 +158,28 @@ namespace alpha_complex {
  * `SimplicialComplexForAlpha::prune_above_filtration()`).
  * In the following example, the value is given by the user as argument of the program.
  *
+ * \section weightedversion Weighted specific version
+ * <b>Requires:</b> \ref eigen &ge; 3.1.0 and \ref cgal &ge; 5.1.0.
+ * For performances reasons, it is advised to use \ref eigen &ge; 3.5.8 and \ref cgal &ge; 5.2.0
```

Comment:
A period is missing at the end of this sentence I think.

Reply:
Fixed on 6f2ec5ee

Reply:
I added a point at the end of the sentence plus a carriage return on 6f2ec5e